pkgs/applications/virtualization/crosvm/default-seccomp-policy-dir.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/src/main.rs b/src/main.rs
+index 81f20a7..481ebd7 100644
+--- a/src/main.rs
++++ b/src/main.rs
+@@ -158,7 +158,9 @@ impl Default for Config {
+             wayland_dmabuf: false,
+             shared_dirs: Vec::new(),
+             sandbox: !cfg!(feature = "default-no-sandbox"),
+-            seccomp_policy_dir: PathBuf::from(SECCOMP_POLICY_DIR),
++            seccomp_policy_dir: PathBuf::from(
++                option_env!("DEFAULT_SECCOMP_POLICY_DIR").unwrap_or(SECCOMP_POLICY_DIR),
```

Comment:
Should this be the other way round so it can still be customised on the command line?

Reply:
It should still be customizable from the command line, this order refers to the compile time option being used over the default path, not over the argument.